README.md

```diff
@@ -267,6 +231,10 @@ Public Key: edpku66KahHGQsthyuHmsYm829xnH6jWXiapkyaNf1HspXx5VKKPSu
 And transfer some tokens:
 ```sh
 $ tezos-client transfer 100 from alice to bob --burn-cap 0.257
+
+#TODO: This gives the following error message, even though the transfer seems to succeed:
+# Fatal error:
+#  transfer simulation failed
```

Comment:
btw, have you tried to originate the contract and check whether it also causes this error?